pkgs/applications/editors/neovim/default.nix

```diff
@@ -31,13 +32,13 @@ let
 in
   stdenv.mkDerivation rec {
     pname = "neovim-unwrapped";
-    version = "0.4.4";
+    version = "0.5.0";
 
     src = fetchFromGitHub {
       owner = "neovim";
       repo = "neovim";
-      rev = "v${version}";
-      sha256 = "11zyj6jvkwas3n6w1ckj3pk6jf81z1g7ngg4smmwm7c27y2a6f2m";
+      rev = "78f0f00cd5f6f09320851e51fe61eb818e595f39";
```

Comment:
Please fetch a tag or change the version to unstable-XXXX-XX-XX.

Reply:
This PR will be updated once `0.5.0` is released. `0.5.0` will be a stable release.

Reply:
We could also maintain 2 versions of neovim and track nightly as well